acme/acme/messages.py

```diff
@@ -87,7 +90,7 @@ def with_code(cls, code, **kwargs):
             raise ValueError("The supplied code: %s is not a known ACME error"
                              " code" % code)
         typ = ERROR_PREFIX + code
-        return cls(typ=typ, **kwargs)
+        return cls(typ=typ, **kwargs)  # type: ignore
```

Comment:
I think we should add a comment for this `type: ignore`.

Reply:
Done.